src/pages/permissions/actions/BulkDeleteIdentitiesBtn.tsx

```diff
@@ -1,27 +1,19 @@
 import { FC, useState } from "react";
-import {
-  ButtonProps,
-  ConfirmationButton,
-  Icon,
-  useNotify,
-} from "@canonical/react-components";
+import { ButtonProps, useNotify } from "@canonical/react-components";
 import type { LxdIdentity } from "types/permissions";
 import { deleteIdentities } from "api/auth-identities";
 import { useQueryClient } from "@tanstack/react-query";
 import { useToastNotification } from "context/toastNotificationProvider";
 import { queryKeys } from "util/queryKeys";
 import { pluralize } from "util/instanceBulkActions";
 import { useIdentityEntitlements } from "util/entitlements/identities";
+import BulkDeleteButton from "components/BulkDeleteButton";
 
 interface Props {
   identities: LxdIdentity[];
-  className?: string;
 }
 
-const BulkDeleteIdentitiesBtn: FC<Props & ButtonProps> = ({
-  identities,
-  className,
-}) => {
+const BulkDeleteIdentitiesBtn: FC<Props & ButtonProps> = ({ identities }) => {
```

Comment:
```suggestion
const BulkDeleteIdentitiesBtn: FC<Props> = ({ identities }) => {
```
Seems to be ignored.